src/k8s-engine/wsl.ts

```diff
@@ -44,9 +44,9 @@ import { ContainerEngine, Settings } from '@/config/settings';
 import { jsonStringifyWithWhiteSpace } from '@/utils/stringify';
 import { getImageProcessor } from '@/k8s-engine/images/imageFactory';
 import { getServerCredentialsPath, ServerState } from '@/main/credentialServer/httpCredentialHelperServer';
-import { getVtunnelConfigPath } from '@/main/networking/vtunnel';
 import { KubeClient } from '@/k8s-engine/client';
 import { showMessageBox } from '@/window';
+import { getVtunnelInstance, getVtunnelConfigPath } from '@/main/networking/vtunnel';
```

Comment:
The imports are out of order (we mainly sort by package path).  Of course, it's not like this file was particularly well sorted in the first place…